styles/helpers/_borders.scss

```diff
@@ -7,6 +7,14 @@
   );
 }
 
+.pseudo-left-border-blue-2px {
+  @include before-left-border(
+    2px,
+    solid,
+    theme('colors.blue.primary')
+  );
+}
+
```

Comment:
There are times (not including here) where we just want a left border of that primary blue. So I added in the helper for it!